docker/base/Dockerfile

```diff
@@ -7,9 +7,11 @@ ENV DEBIAN_FRONTEND=noninteractive
 COPY apt-conf /etc/apt/apt.conf.d/
 
 RUN apt-get update && \
-    apt-get install autoconf \
-                    python3-dev \
-                    python-dev \
-                    libiberty-dev \
-                    build-essential \
-                    make
+    apt-get install -y autoconf \
+                       python3-dev \
+                       python-dev \
```

Comment:
I doubt this line is needed. The correct deps should be installed when python3-dev is installed.

Reply:
@andy5995 thanks for the input, this PR is originally for the Dockerfile basic improvement, using the most common best practices, so other deep-dive improvement were not considered to be together at that moment. Just FYI ;)

Reply:
@PeterDaveHello Understood :)